src/Settings.php

```diff
@@ -145,6 +145,11 @@ public static function parseArguments(array $arguments)
         $arguments = new ArrayIterator(array_slice($arguments, 1));
         $settings = new self;
 
+        //use the currently invoked php as the default if possible
```

Comment:
Just small code style: Can you please put space after `//` and start commend with big letter?

Reply:
Done